web/src/frontend/src/utils.ts

```diff
@@ -16,3 +16,42 @@ export function getApiBaseUrl(): string {
 }
 
 export const MED_CREDENTIALS: RequestInit = { credentials: "same-origin" };
+
+/* For å snakke mot sendsoknad app når den kjører lokalt */
+const API_BASE_URL = "http://localhost:8181/sendsoknad";
```

Comment:
Ingvild hadde litt kode for det her du kanskje bør se på

Reply:
Når mock backenden er klar, så skal man bruke funksjonen til Ingvild getApiBaseUrl().